packages/app/src/app/store/providers/Router.js

```diff
@@ -10,6 +10,9 @@ export default Provider({
   redirectToNewSandbox() {
     history.push('/s/new');
   },
+  redirectToSandboxWizard() {
+    history.replace('/s');
```

Comment:
Hey @pshrmn , can you please make it `/s/`, to prevent the redirect?

Reply:
```suggestion
    history.replace('/s/');
```